### PR TITLE
fix: use `view.rows`

### PR DESCRIPTION
**Pull request by jo**

### CODE REVIEW

File: lib/compose-plugin.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 module.exports = composePlugin
 
-var _ = require('lodash')
```
Comment (peritus) [inline lib/compose-plugin.js]: Unrelated!

Comment (jo) [inline lib/compose-plugin.js]: no, I made it unused :)

Comment (jo) [inline lib/compose-plugin.js]: in the pr you merged before
